### ISSUE TITLE
Upgrade to Tauri v2

### ISSUE THREAD

**Issue by minthemiddle** [minthemiddle/Quick-Capture#39]
Tauri 2.0 Stable was released in October 24. ([Release notes](https://tauri.app/blog/tauri-20/)).  
Documentation is better than in v1 and v2 beta.  
Upgrading might e.g. help with getting better builds via Github Actions.

**Comment by minthemiddle** [comment]
I'm just following the `Cargo` instructions for upgrading from here:
https://tauri.app/start/migrate/from-tauri-1/

**Comment by minthemiddle** [comment]
Path not showing after upgrade.
It's a toggle (either "Daily" or "Standalone").
Both have a CSS `hidden` class, but just one should have, matching the selected entry mode. 

![image](https://github.com/user-attachments/assets/b8311a21-45cc-4419-8bf7-6bf664bf6ce0)

There's only 1 error in the console and it's probably not related:

```
[Error] TypeError: Right side of assignment cannot be destructured
	Module Code (main.js:1)
```

**Comment by minthemiddle** [comment]
There's now an official recommendation for Github Action that I also implemented:
https://tauri.app/distribute/pipelines/github/

**Comment by minthemiddle** [comment]
Solved via 52728e6abca6bf41ef8aba17a66145e618ccc6cf
